ocs_ci/helpers/performance_lib.py

```diff
@@ -86,3 +89,111 @@ def run_command(cmd, timeout=600, out_format="string", **kwargs):
         output = output.split("\n")  # convert output to list
         output.pop()  # remove last empty element from the list
     return output
+
+
+def run_oc_command(cmd, namespace):
+    """
+    Running an 'oc' command
+
+    Args:
+        cmd (str): the command to run
+        namespace (str): the namespace where to run the command
+
+    Returns:
+        list : the results of the command as list of lines
+
+    """
+
+    cluster_dir_kubeconfig = os.path.join(
+        config.ENV_DATA["cluster_path"], config.RUN.get("kubeconfig_location")
+    )
+    if os.getenv("KUBECONFIG"):
+        kubeconfig = os.getenv("KUBECONFIG")
+    elif os.path.exists(cluster_dir_kubeconfig):
+        kubeconfig = cluster_dir_kubeconfig
+    else:
+        kubeconfig = None
+
+    command = f"oc --kubeconfig {kubeconfig} -n {namespace} {cmd}"
+    return run_command(command, out_format="list")
+
+
+def get_log_names(interface):
+    """
+    Finds names for log files pods in which logs for clone creation are located
+    For CephFS: 2 pods that start with "csi-cephfsplugin-provisioner" prefix
+    For RBD: 2 pods that start with "csi-rbdplugin-provisioner" prefix
+
+    """
+    log_names = []
+
+    pods = run_oc_command(cmd="get pod", namespace="openshift-storage")
+
+    if "Error in command" in pods:
+        raise Exception("Can not get csi controller pod")
+
+    provisioning_name = "csi-cephfsplugin-provisioner"
+    if interface == constants.CEPHBLOCKPOOL:
+        provisioning_name = "csi-rbdplugin-provisioner"
+
+    for line in pods:
+        if provisioning_name in line:
+            log_names.append(line.split()[0])
+
+    logger.info(f"The logs pods are : {log_names}")
+    return log_names
+
+
+def measure_pvc_creation_time(interface, pvc_name, start_time):
```

Comment:
@Avilir  The difference between my function in performance_lib.py and the ocs_ci.helpers.helpers.measure_pvc_creation_time() is that we are not going over all the log file but starting to search from start time parameter. 
Also their function does not take into consideration the fact that somoetimes there are multiple start time and multiple end time records in the log. 
I am also usiung this function in the multiple clones test, in which the time of the test is critical. 

Therefore I need this function as it is written in the performance_lib to make sure that 
1) we are not going over all the log file to search for the records 
2) We calculate the creation time as the difference between the last "end time" appearance and the first "start time" appearance. And the old helpers function does not implement that. 

I did not want to change the function in ocs_ci.helpers , since there might be other tests depending on it. 
After this PR is merged, I will also take those functions from multi clone test ( in order not to duplicate code). 
I'll do it in a separate PR since this PR is already big enough.